src/core/client/stream/tabs/Live/LivePostCommentFormContainer.tsx

```diff
@@ -89,10 +89,10 @@ export const LivePostCommentFormContainer: FunctionComponent<Props> = ({
   const [nudge, setNudge] = useState(true);
   const [submitStatus, setSubmitStatus] = useState<SubmitStatus | null>(null);
 
-  const [draft = "", setDraft, initialDraft] = usePersistedState<string>(
+  const [draft = "", setDraft, initialDraft] = usePersistedSessionState<string>(
```

Comment:
Doing these here as well as in the other PR because otherwise it won't compile and we won't be able to merge.